tapalcatl_server/handler.go

```diff
@@ -25,23 +28,24 @@ type BufferManager interface {
 	Put(*bytes.Buffer)
 }
 
-// the logState structures and string generation serve to emit a single log entry line
+// the reqState structures and string generation serve to emit a single log entry line
 // a log parser will pick this up and use it to persist metrics
 // the string functions here are specific to the format used and should be updated with care
 
-type logResponseState int32
+type reqResponseState int32
 
 const (
-	ResponseState_Nil logResponseState = iota
+	ResponseState_Nil reqResponseState = iota
```

Comment:
Do we need the "nil" response state? As far as I can tell, it's not used anywhere in the code?